### DISCUSSION TITLE
How to open dropdown menu with Angular ?

### DISCUSSION THREAD

**Discussion by Nico7522** [twbs/bootstrap#39048] [General] [answered]
Hello,

I want to add a simple dropdown menu to my app but when i click and want to show the menu nothing appear.
I looked for a solutation and tried several things but nothing is working. 

Here my angular.json :

`            "styles": [
              "src/styles.css",
              "node_modules/bootstrap/dist/css/bootstrap.css",
              "node_modules/bootstrap/dist/css/bootstrap.min.css"
             

            ],
            "scripts": [
              "node_modules/jquery/dist/jquery.min.js",
              "node_modules/bootstrap/dist/js/bootstrap.bundle.min.js",
              "node_modules/popper.js/dist/umd/popper.min.js",
              "node_modules/bootstrap/dist/js/bootstrap.min.js"
            ]`

I think I just missing something but I can get why. Thanks a lot guys.

**Comment by gerardnico** [comment, marked as answer]
Dropdowns are managed by popper.
You therefore need to register them via javascript.

The easiest way is to add data attributes and let the Bootstrap script do it. See:
https://getbootstrap.com/docs/5.3/components/dropdowns/#via-data-attributes

Note: if I'm right, `bootstrap.bundle.min.js` is the bundle of `popper.min.js` and `bootstrap.min.js`. You should have one or the other but not the two.

**Reply by Nico7522** [reply]
Yes i finaly solved by just add this 
     "scripts": [
              "node_modules/jquery/dist/jquery.min.js",
              "node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"
            ]
You're right, thanks !
